src/main/kotlin/persistence/data/Api.kt

```diff
@@ -0,0 +1,24 @@
+package persistence.data
+
+import data.Language
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+class Api {
```

Comment:
Api is too generic, we could have more than one API

Reply:
ok. changed to Door43Api and renamed the interface Door43Api to Door43ApiInterface